src/Form/Control/Dropdown.php

```diff
@@ -159,8 +159,11 @@ protected function jsDropdown($when = false, $action = null): JsExpressionable
     protected function jsRenderDropdown(): JsExpressionable
```

Comment:
and what about https://github.com/atk4/ui/pull/2063/commits/6fcc1f225c0b2f0095d8d4106b1ed92ba6b37a2e?

Reply:
> and what about [6fcc1f2](https://github.com/atk4/ui/commit/6fcc1f225c0b2f0095d8d4106b1ed92ba6b37a2e)?

According to the Human Inteface Guidlines (in most versions) a radio-button group can not be deselected as well as only one option should be selectable. If you want an unselected radio-button group, you should really use checkboxes instead.

Check https://uxplanet.org/radio-buttons-ux-design-588e5c0a50dc for example.

Reply:
> Also please clarify the motivation of this PR.

I added it to the initial comment.

Reply:
I am ok to allow to disable the "clearable" in opt-in fashion.